Composer/packages/client/src/components/CreationFlow/FileSelector.tsx

```diff
@@ -480,6 +480,13 @@ export const FileSelector: React.FC<FileSelectorProps> = (props) => {
       setCurrentPath(value as string);
     }
   };
+
+  const updatePathPending = (option?: IComboBoxOption, index?: number, value?: string) => {
+    if (!option && value) {
+      setCurrentPath(value as string);
```

Comment:
I think here is a good opportunity to test the path and if it's invalid show an error or something.